resources/js/Component/CardAlbum.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <section>
+        <div class="box is-dark">
+            <div class="container has-text-centered is-dark">
+                <p class="title is-4">Vacances a Rome</p>
+                <p class="subtitle is-6">Bedric Sila</p>
+            </div>
+        </div>
+    </section>
+</template>
+<style scoped lang="scss">
+.box {
+    background: linear-gradient(rgba(0, 0, 0, 0.35), rgba(0, 0, 0, 0.35)),
+        url("https://picsum.photos/600/400") no-repeat center center;
```

Comment:
Quick question 🤔  When it will be the time to swap the placeholder to use the images on the server, How are you going to do it ?
Because I see no variable coming from outside the `CardAlbum` component.
I did a quick search and it seems you can use variable in SCSS with Vue : https://shayneo.com/blog/binding-css-variables-with-vue/ maybe you can find more resources by yourself.